test/mason/init/masonInteractive.compopts

```diff
@@ -0,0 +1 @@
+-M ../../tools/mason/
```

Comment:
This can now be a `COMPOPTS` file that symlinks `../COMPOPTS` (I have updated this)